### PR TITLE
[common-plugins] add individual text alignment buttons

### PR DESCRIPTION
**Pull request by noamtwix**

### CODE REVIEW

File: packages/ricos-common-plugins/web/src/text-align/text-align.ts [hunk 1]
```diff
@@ -75,5 +76,77 @@ export const pluginTextAlignment: TiptapEditorPlugin = {
...
+        active: RESOLVERS_IDS.IS_TEXT_ALIGN_RIGHT,
+        disabled: RESOLVERS_IDS.IS_NODE_SELECTED,
+      },
+      command: (editorCommands: EditorCommands) => () => {
+        editorCommands.setTextAlignment('right');
+        return true;
+      },
+    },
+    {
+      id: 'JUSTIFY',
```
Comment (maimonav) [inline packages/ricos-common-plugins/web/src/text-align/text-align.ts:134]: `ALIGN_JUSTIFY`, just for consistency

Comment (noamtwix) [reply packages/ricos-common-plugins/web/src/text-align/text-align.ts:134]: this is not consistent [here](https://github.com/wix/ricos/blob/master/packages/editor-common/web/src/consts.ts#L100:L100).
so it's a problem

File: packages/toolbars-v3/web/src/resolvers/tiptapResolvers.ts [hunk 2]
```diff
@@ -143,15 +144,73 @@ export const isTextContainsSpoilerResolver = TiptapContentResolver.create(
...
+  if (textStyles[0]) {
+    currentTextStyle = textStyles[0].toLowerCase();
+  }
+  if (currentTextStyle === 'auto') {
+    const selectedFirstNodeText = doc.nodeAt(from - 1)?.textContent;
+    const selectedFirstNodeDirection = getTextDirection(selectedFirstNodeText);
+    currentTextStyle = selectedFirstNodeDirection === 'rtl' ? 'right' : 'left';
+  }
+  return currentTextStyle as TextAlignment;
+};
```
Comment (maimonav) [inline packages/toolbars-v3/web/src/resolvers/tiptapResolvers.ts:172]: Can't we add the changes in the `activeTextStyles` query?

File: packages/ricos-common-plugins/web/src/text-align/text-align.ts (hunk 1 above)
Comment (maimonav) [reply packages/ricos-common-plugins/web/src/text-align/text-align.ts:134]: @lxgreen refactored all config id's, making them consistent throughout the entire plugins.
We should define the keys, and convert them if we have to legacy values.
Please take a look and let us know what is the correct pattern for such configs.

File: packages/ricos-common-plugins/web/src/text-align/text-align.ts [hunk 3]
```diff
@@ -75,5 +76,77 @@ export const pluginTextAlignment: TiptapEditorPlugin = {
         Component: AlignmentPanelComponent,
       },
     },
+    {
```
Comment (maimonav) [inline packages/ricos-common-plugins/web/src/text-align/text-align.ts:79]: Change to 'TextAlignment.ALIGNMENT'/'TextAlignment.LEFT'/'TextAlignment.CENTER'/'TextAlignment.RIGHT'/'TextAlignment.JUSTIFY'

File: packages/ricos-plugins/web/src/toolbar-utils/toToolbarItemsConfig.ts [hunk 4]
```diff
@@ -21,6 +21,10 @@ const cleanTitleIfNeeded = (
 const toSchemaBasedId = (formattingButtonId: string): string => {
   const compatibleIds = [
     'ALIGNMENT',
+    'ALIGN_LEFT',
+    'ALIGN_CENTER',
+    'ALIGN_RIGHT',
+    'JUSTIFY',
```
Comment (maimonav) [inline packages/ricos-plugins/web/src/toolbar-utils/toToolbarItemsConfig.ts:27]: Change all id's to legacy api

Comment (maimonav) [reply packages/ricos-plugins/web/src/toolbar-utils/toToolbarItemsConfig.ts:27]: Line 42, mapping alignment buttons

Comment (maimonav) [reply packages/ricos-plugins/web/src/toolbar-utils/toToolbarItemsConfig.ts:27]: Add opposite conversion to `plugin-text-button.ts` to all "new" alignment buttons

**Pull request merged**